### PR TITLE
cmd: do not require userns for "version"

### PR DESCRIPTION
**Pull request by giuseppe**
Closes: https://github.com/containers/podman/issues/17657

#### Does this PR introduce a user-facing change?

```release-note
podman version does not join the rootless user namespace
```

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
@giuseppe looks like you have some gofmt issues.

**Review by rhatdan — APPROVED** [review]
LGTM

**Review by Luap99 — COMMENTED** [review]
code change looks good but I think the ParentNSRequired comment should be improved

File: cmd/podman/registry/config.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ const (
 	// NoMoveProcess used as cobra.Annotation when command doesn't need Podman to be moved to a separate cgroup
 	NoMoveProcess = "NoMoveProcess"
 
-	// ParentNSRequired used as cobra.Annotation when command requires root access
+	// ParentNSRequired used as cobra.Annotation when command must run in the  requires root access
```
[the remark this answers is not in the document]
Comment (giuseppe) [reply cmd/podman/registry/config.go:20]: that is better, tweaked to:

```
	// ParentNSRequired used as cobra.Annotation when a command should not be run in the podman rootless user namespace, also requires updates in `pkg/rootless/rootless_linux.c` in function `can_use_shortcut()` to exclude the command name there.
```

**Review by Luap99 — COMMENTED** [review]
LGTM

**Pull request merged**